SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/SpatialRPCService.cpp

```diff
@@ -376,20 +379,13 @@ EPushRPCResult SpatialRPCService::PushRPCInternal(const Worker_EntityId EntityId
 
 		RPCStore.LastSentRPCIds.Add(EntityType, NewRPCId);
 	}
+	// Reliable RPC Overflowed
 	else
 	{
-		// Overflowed
-		if (RPCRingBufferUtils::ShouldQueueOverflowed(Type))
-		{
-			return EPushRPCResult::QueueOverflowed;
-		}
-		else
-		{
-			return EPushRPCResult::DropOverflowed;
-		}
+		return EPushRPCResult::QueueOverflowed;
 	}
 
-	return EPushRPCResult::Success;
+	return NotOverflowing ? EPushRPCResult::Success : EPushRPCResult::DropOverflowed;
```

Comment:
Could that cause an incorrect log to be emitted somewhere else ? Like "RPC X has been dropped", when it is "RPC Y has been overwritten " ?

Reply:
Indeed. I think I'll put the overwritten RPC log from warning to verbose/regular log when it's an unreliable one, and remove the "RPX X has been dropped" log in the unreliable case.